projects/ngx-charts-on-fhir/src/lib/fhir-chart-layout/fhir-chart-layout.component.html

```diff
@@ -5,7 +5,7 @@
       <div class="layout-sidebar-content">
         <patient-browser *ngIf="toolbar.includes('patients')" [hidden]="active !== 'patients'"></patient-browser>
         <data-layer-browser *ngIf="toolbar.includes('browser')" [hidden]="active !== 'browser'"></data-layer-browser>
-        <data-layer-list *ngIf="toolbar.includes('options')" [hidden]="active !== 'options'"></data-layer-list>
+        <data-layer-list *ngIf="toolbar.includes('options')" [hidden]="active !== 'options'" [hideRemoveLayerButton]="hideRemoveLayerButton"></data-layer-list>
```

Comment:
fhir-chart-layout can decide to hide the button based on whether the browser panel is available:
```suggestion
        <data-layer-list *ngIf="toolbar.includes('options')" [hidden]="active !== 'options'" [hideRemoveLayerButton]="!toolbar.includes('browser')"></data-layer-list>
```
That way the button is always hidden when there's no way to add layers and the app doesn't have to configure the button explicitly.